tlsconfig/config.go

```diff
@@ -83,6 +84,22 @@ func certPool(caFile string) (*x509.CertPool, error) {
 	return certPool, nil
 }
 
+// minVersion returns the minimum TLS version matching 'version', the MinVersion string.
+func minVersion(version string) (uint16, error) {
+	allVersions := map[string]uint16{
+		"VersionSSL30": tls.VersionSSL30,
```

Comment:
Think of this as, this would be the string displayed in the docker config at `"MinVersion": "VersionSSL30"`